binaries/cuprated/src/config/default.rs

```diff
@@ -0,0 +1,69 @@
+use std::{
+    io::Write,
+    path::{Path, PathBuf},
+};
+
+use cuprate_helper::fs::{CUPRATE_BLOCKCHAIN_DIR, CUPRATE_CACHE_DIR, CUPRATE_TXPOOL_DIR};
+
+use crate::config::DEFAULT_CONFIG_FILE_NAME;
+
+/// Creates a config file which will be named [`DEFAULT_CONFIG_FILE_NAME`] in the directory given in [`Path`].
+///
+/// This will always terminate the program, on success and failure.
+pub fn create_default_config_file(path: &Path) -> ! {
+    let config_file = path.join(DEFAULT_CONFIG_FILE_NAME);
+
+    tracing::info!("Attempting to create new config file here: {config_file:?}");
+
+    let mut file = match std::fs::OpenOptions::new()
+        .write(true)
+        .create_new(true)
+        .open(&config_file)
+    {
+        Ok(file) => file,
+        Err(e) => {
+            tracing::error!("Failed to create config file, got error: {e}");
+            std::process::exit(1);
+        }
+    };
+
+    let config = generate_config_text();
+    file.write_all(config.as_bytes()).unwrap();
+
+    std::process::exit(0);
+}
+
+/// Generates the text of the default config file.
+fn generate_config_text() -> String {
+    format!(
+        include_str!("Cuprate.toml"),
+        cache = CUPRATE_CACHE_DIR.to_string_lossy(),
+        txpool = CUPRATE_TXPOOL_DIR.to_string_lossy(),
+        blockchain = CUPRATE_BLOCKCHAIN_DIR.to_string_lossy()
+    )
+}
+
+#[cfg(test)]
+mod tests {
+    use crate::config::{default::generate_config_text, Config};
+
+    #[test]
+    fn generate_config_text_covers_all_values() {
```

Comment:
Some tests to make sure the generated config file is correct and contains all values.